### ISSUE TITLE
'hr_data' tensor size question during training

### ISSUE THREAD

**Issue by zzhan127** [skycrapers/TecoGAN-PyTorch#5]
Hello,

I have a quick question regarding the tensor size of hr_data in vsrgan network. During loss calculation, I noticed that the size of hr_data is (4, 19, 3, 128, 128). 
The 4 is batch size, 3 is channels, 128 is crop size. But I am not sure what is 19 here?

Thank you!

**Comment by marwahmh** [comment]
did you know the answer?
